src/commands/repoRelease.js

```diff
@@ -24,44 +24,42 @@ const program = new commander.Command();
 const cliOptions = require('./cliOptions.js');
 
 program
-    .name('taoRelease createRelease')
+    .name('taoRelease ghRelease')
     .usage('[options]')
     .option(...cliOptions.debug)
+    .option(...cliOptions.releaseVersion)
     // options with defaults
     .option(...cliOptions.baseBranch)
     .option(...cliOptions.branchPrefix)
     .option(...cliOptions.origin)
     .option(...cliOptions.releaseBranch)
-    .option(...cliOptions.wwwUser)
     // options which fall back to user prompts if undefined
-    .option(...cliOptions.pathToTao)
-    .option(...cliOptions.extensionToRelease)
-    .option(...cliOptions.versionToRelease)
-    .option(...cliOptions.updateTranslations)
     .option(...cliOptions.releaseComment)
     .parse(process.argv);
 
 if (program.debug) {
     log.info(program.opts());
 }
 
-const release = require('../release.js')({ ...program.opts(), subjectType: 'extension' });
+const release = require('../release.js')({ ...program.opts(), subjectType: 'repo' });
 
-async function createRelease() {
+async function npmRelease() {
     try {
-        log.title('TAO Extension Release: createRelease');
+        log.title('Release a repository');
```

Comment:
This command list doesn't contain `updateVersion()`, is it intentional?

Reply:
Also I don't know if you considered de-duplicating the common commands (everything from `loadConfig()` to `createReleasingBranch()`) which appear in exactly the same sequence in 3 places. Or maybe there's enough abstraction already in this project. It could be a wasted effort considering that the command lists are already very readable.

Reply:
Yes, it's intended for PHP or GO repositories (not tao extension) that only need a tag to be created: no build, no version file, etc.